lib/src/models/client-config.ts

```diff
@@ -49,7 +46,24 @@ export interface StrictAuthClientConfig {
      * Note that if this is set to false, then the needed endpoints should be provided through
      * the `endpoints` property.
      */
-    sendWellKnownEndpointRequest?: boolean;
+     environment?: SERVER_ENVIRONMENTS;
```

Comment:
Can we remove this additional space here?

Reply:
The specified `environment` field is removed as the approach has been changed.